app-nextjs-starter/src/Story.ts

```diff
@@ -30,8 +24,7 @@ export const isStory = (data: unknown): data is Story =>
   typeof data.slug === 'string' &&
   'updated_at' in data &&
   typeof data.updated_at === 'string' &&
-  'last_author' in data &&
-  isLastAuthor(data.last_author)
+  'last_author' in data
```

Comment:
What's the default value of last_author here? Is it empty string or undefined? If undefined, this condition will fail, right?

Reply:
ahh... it's `null`. I think this condition may give false impression that `last_author` exists in the data. What if we remove the condition at all?